stopcovid/send_sms.py

```diff
@@ -0,0 +1,19 @@
+import json
+from time import sleep
+from stopcovid.clients import twilio, kinesis
+
+
+def _get_delay_seconds(record):
+    return int(record["messageAttributes"].get("delay_seconds", {}).get("stringValue", "0"))
+
+
+def _send_message_to_twilio(record):
+    message = json.loads(record["body"])
+    sleep(_get_delay_seconds(record))
+    return twilio.send_message(message["To"], message["Body"])
+
+
+def send_sms(event, context):
+    twilio_responses = [_send_message_to_twilio(record) for record in event["Records"]]
+    kinesis.publish_log_outbound_sms(twilio_responses)
```

Comment:
note that, if this fails, we'll end up double sending. I'd probably log+ignore any error here.

Reply:
Good catch. Ill wrap it in a try except